test/ng/directive/ngOptionsSpec.js

```diff
@@ -2143,6 +2143,26 @@ describe('ngOptions', function() {
       expect(scope.selected).toEqual([]);
     });
 
+    it('should be possible to specify a value for empty options', function() {
+      var html = '<select ng-model="someModel" ng-options="c for c in choices" ng-empty-value="someVar">' +
```

Comment:
If I use createSingleSelect, how do I give it an ng-empty-value attribute? I was reluctant to change such a heavily used function for just the one test.